src/lib/components/forms/form/form-components/slider/slider.component.spec.ts

```diff
@@ -0,0 +1,126 @@
+import { TerraPlacementEnum } from '../../../../../helpers';
+
+describe('SliderComponent', () => {
+    let fixture: ComponentFixture<SliderComponent>;
+    let component: SliderComponent;
+    let loader: HarnessLoader;
+    let input: MatSliderHarness;
+
+    beforeEach(async () => {
+        TestBed.configureTestingModule({
+            imports: [MatFormFieldModule, MatSliderModule, NoopAnimationsModule, FormsModule],
+            declarations: [SliderComponent, MockTooltipDirective]
+        });
+
+        fixture = TestBed.createComponent(SliderComponent);
+        component = fixture.componentInstance;
+        loader = TestbedHarnessEnvironment.loader(fixture);
+
+        fixture.detectChanges();
+
+        input = await loader.getHarness(MatSliderHarness);
+    });
+
+    it('should create', async () => {
+        expect(component).toBeTruthy();
+        expect(input).toBeTruthy();
+    });
+
+    it('should disable the input when #isDisabled is set', async () => {
+        expect(await input.isDisabled()).toBe(false);
+        component.isDisabled = true;
+        fixture.detectChanges();
+
+        expect(await input.isDisabled()).toBe(true);
+    });
+
+    it('should have #name as label of the input', async () => {
+        const formField: MatFormFieldHarness = await loader.getHarness(MatFormFieldHarness);
+        expect(await formField.getLabel()).toBe('');
+
+        component.name = 'My Label';
+        fixture.detectChanges();
+        expect(await formField.getLabel()).toBe(component.name);
+    });
+
+    describe('with tooltip', () => {
+        let tooltip: MockTooltipDirective;
+        const testToolTipCaption: string = 'testToolTip';
+
+        beforeEach(() => {
+            tooltip = fixture.debugElement.query(By.directive(MockTooltipDirective)).injector.get(MockTooltipDirective);
+        });
+
+        it('should set tooltip placement according to #tooltipPlacement', () => {
+            expect(tooltip.placement).toBe(TerraPlacementEnum.TOP);
+            component.tooltipPlacement = TerraPlacementEnum.RIGHT;
+            fixture.detectChanges();
+            expect(tooltip.placement).toBe(TerraPlacementEnum.RIGHT);
+        });
+
+        it('should set tooltiptext according to #tooltipText', () => {
+            expect(tooltip.tcTooltip).toBeFalsy();
+            component.tooltipText = testToolTipCaption;
+            fixture.detectChanges();
+            expect(tooltip.tcTooltip).toBe(testToolTipCaption);
+        });
+    });
+
+    it('should call registered onTouchedCallback when input was blurred', async () => {
+        const spy: jasmine.Spy = jasmine.createSpy('onTouchedCallback');
+        component.registerOnTouched(spy);
+        await input.blur();
+
+        expect(spy).toHaveBeenCalled();
+    });
+
+    it('should set min/max values', async () => {
+        const minValue: number = 10;
+        const maxValue: number = 100;
+
+        component.min = minValue;
```

Comment:
Da fehlt noch was ;)
```suggestion
        component.min = minValue;
        component.max = maxValue;
```